### PR TITLE
Stack 299, A10_write_memory example

### PR DESCRIPTION
**Pull request by swapkashid**

### CODE REVIEW

File: examples/a10_shutdown/a10_shutdown.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - name: a10_shutdown vthunder instance
+    a10_shutdown:
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      at: 1
+      time: "15.00" # Time in HH:MM format
+
```
Comment (hthompson6) [inline examples/a10_shutdown/a10_shutdown.yml]: Remove extra whitespace.

File: examples/a10_reload/a10_reload_device.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - name: a10_reload specific device
+    a10_reload:
+      state: "present"
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }} "
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      device: 1
+
```
Comment (hthompson6) [inline examples/a10_reload/a10_reload_device.yml]: Remove extra whitespace.

File: examples/a10_reload/a10_reload_all.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  tasks:
+  - name: a10_reload all
+    a10_reload:
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      all: 1
+
```
Comment (hthompson6) [inline examples/a10_reload/a10_reload_all.yml]: Remove extra whitespace.

File: examples/a10_write_memory/a10_write_memory.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  - name: a10_write_memory instance
+    a10_write_memory:
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}}"
+      partition: "shared"
+      destination: "primary"
+
```
Comment (hthompson6) [inline examples/a10_write_memory/a10_write_memory.yml]: Remove extra whitespace.

File: examples/a10_timezone/a10_timezone.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      timezone_index_cfg: 
+        timezone-index : "Asia/Calcutta"
+        nodst : true
+
+
+        # Supported Values for Timezone
```
Comment (hthompson6) [inline examples/a10_timezone/a10_timezone.yml]: This commend should be removed. They should refer to the AXAPI documentation for supported values.

File: examples/a10_shutdown/a10_shutdown.yml (hunk 1 above)
Comment (swapkashid) [reply examples/a10_shutdown/a10_shutdown.yml]: Resolved

File: examples/a10_reload/a10_reload_device.yml (hunk 2 above)
Comment (swapkashid) [reply examples/a10_reload/a10_reload_device.yml]: Resolved

File: examples/a10_reload/a10_reload_all.yml (hunk 3 above)
Comment (swapkashid) [reply examples/a10_reload/a10_reload_all.yml]: Resolved

File: examples/a10_write_memory/a10_write_memory.yml (hunk 4 above)
Comment (swapkashid) [reply examples/a10_write_memory/a10_write_memory.yml]: Resolved

File: examples/a10_timezone/a10_timezone.yml (hunk 5 above)
Comment (swapkashid) [reply examples/a10_timezone/a10_timezone.yml]: Resolved

**Pull request merged**